src/sidebar/store/modules/test/groups-test.js

```diff
@@ -108,4 +139,135 @@ describe('sidebar.store.modules.groups', () => {
       assert.equal(store.focusedGroupId(), privateGroup.id);
     });
   });
+
+  describe('getFeaturedGroups', () => {
+    [
+      {
+        description:
+          'If logged in and there is a restricted group the user is not a member of, show it in `Featured Groups`.',
+        isLoggedIn: true,
+        filteredGroups: [restrictedGroup],
```

Comment:
It wasn't obvious to me what `filteredGroups` meant at first and how it related to `expectedGroups`. I think `allGroups` and `expectedFeaturedGroups` would have been clearer names.